Create LICENSE

stevepiercy opened the pull request:
See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository.

See https://github.com/plone/.github/community to see current community health status.

stevepiercy committed:
Create LICENSE

See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

dataflake commented:
According to https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file a license file is not one of the supported files for the `.github` community repository, so to me it feels wrong to use that as a set of suggestions for licenses because it apparently won't be applied to new repositories. 

Also, the status page at  https://github.com/plone/.github/community may not be appropriate for this `.github` repository at all because it's a special one-off and not a common repository.

It won't hurt to add a license here, but my reading of https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository suggests that you should add a specific license that governs this one repository, not a set of suggestions for new repositories.

stevepiercy committed:
Update LICENSE

stevepiercy committed:
There can be only *one* license

stevepiercy commented:
Thanks @dataflake for your feedback. There happens to be an old open issue that I updated to address that at https://github.com/plone/plone.org/issues/148. I also updated this PR accordingly. Could I get one more pass from you? Thank you!
